src/config.ts

```diff
@@ -1,3 +1,25 @@
 export const TCR_CONTRACT_ADDRESS = process.env.TCR_CONTRACT_ADDRESS
 export const TCR_LIST_ID = Number(process.env.TCR_LIST_ID) || 0
 export const TCR_CACHE_TIME = 5 * 60 * 1000
+export const TOKEN_OVERRIDES = {
+  1: {
+    // Deprecated sUSD proxy contract
+    '0x57ab1e02fee23774580c119740129eac7081e9d3': {
+      forceAddressDisplay: true,
```

Comment:
why don't you add add the name and symbols?
```json
name: 'Synthetics sUSD (deprecated) (this is rinkeby, so not really)',
symbol: 'sUSD-old',
```

Reply:
Because they aren't used at all. I kept it for Rinkeby only for reference of what can be done if needed.